klaus/templates/repo_list.html

```diff
@@ -36,4 +36,17 @@ <h2>
   {% endfor %}
 </ul>
 
+{% if invalid_repos is defined and invalid_repos|length>0 %}
```

Comment:
Is it ever undefined?

Reply:
Not in the current code. I will remove the check.